File: Lista 2 sala/Python/atividade10.py
# 10)- Faça um algoritmo que receba como entrada o mês (de 1 a 12) e retorne o nome
# do respectivo mês. (Utilizar caso/seja)

cursos = ['Engenharia', 'Edificações', 'Sistemas Elétricos', 'Turismo', 'Análise de sistemas']

numero_curso = int(input("Qual curso voce gostaria de acessar? (1 a 5): "))

if 1<= numero_curso <=5:
    print(f"O curso selecionado é {cursos[numero_curso - 1]}")
else:
    print("Curso não encontrado")

File: Lista 2 sala/C/atividade10.c
#include <stdio.h>

int main() {
    int numero_curso;
    const char *cursos[] = {
        "Engenharia",
        "Edificações",
        "Sistemas Elétricos",
        "Turismo",
        "Análise de sistemas"
    };
    int total_cursos = sizeof(cursos) / sizeof(cursos[0]);

    printf("Qual curso voce gostaria de acessar? (1 a %d): ", total_cursos);
    scanf("%d", &numero_curso);

    if (numero_curso >= 1 && numero_curso <= total_cursos) {
        printf("O curso selecionado é %s\n", cursos[numero_curso - 1]);
    } else {
        printf("Curso não encontrado\n");
    }

    return 0;
}
